graphstorm-processing/tests/test_dist_heterogenous_loader.py

```diff
@@ -264,15 +263,8 @@ def test_load_dist_hgl_without_labels(dghl_loader_no_label: DistHeterogeneousGra
 
     graphinfo_updates = {
         "nfeat_size": {},
-        "task_type": "link_predict",
-        "etype_label": [
-            "movie:included_in:genre",
-            "genre:included_in-rev:movie",
-            "user:rated:movie",
-            "movie:rated-rev:user",
-            "director:directed:movie",
-            "movie:directed-rev:director",
-        ],
+        "task_type": "link_prediction",
```

Comment:
I think we should keep the original test and add a new test with `"etype_label": [],`

Reply:
Could you clarify what each test would do? The test (`test_load_dist_hgl_without_labels`) checks the default behavior when no label is provided. Before we would generate masks for all edge types, leading to all types having "labels". Since we changed this default behavior we needed to change the test as well.

Reply:
I c